### ISSUE TITLE
Flaw in the return type 

### ISSUE THREAD

**Issue by mitasych** [lagdo/symfony-facades#3]
Hi!
First of all, I want to thank you for the excellent implementation of system facades for symphony!
I found a small bug, not even a bug, but a shortcoming:

```
1x: Method "Symfony\Component\HttpKernel\Bundle\Bundle::boot()" might add "void" as a native return type declaration in the future. Do the same in child class "Lagdo\Symfony\Facades\FacadesBundle" now to avoid errors or add an explicit @return annotation to suppress this message.

  1x: Method "Symfony\Component\HttpKernel\Bundle\Bundle::build()" might add "void" as a native return type declaration in the future. Do the same in child class "Lagdo\Symfony\Facades\FacadesBundle" now to avoid errors or add an explicit @return annotation to suppress this message.

  1x: Method "Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface::process()" might add "void" as a native return type declaration in the future. Do the same in implementation "Lagdo\Symfony\Facades\DependencyInjection\Compiler\CompilerPass" now to avoid errors or add an explicit @return annotation to suppress this message.
```
in Symfony 7.1 context.

I would be grateful if you fix it.

**Comment by feuzeu** [comment]
Hi,

That issue cannot be fixed on the 2.3.* release of this package, because it has to be compatible with older Symfony and PHP versions.
I've released a new version 3.0.0 for Symfony >=7.0 only.

**Comment by mitasych** [comment]
@feuzeu Excellent! Thanks a lot!
